build.sbt

```diff
@@ -1,22 +1,20 @@
 val clueVersion            = "0.23.1"
 val lucumaCoreVersion2     = "0.45.0"
-val lucumaCoreVersion      = "0.51.0"
+val lucumaCoreVersion      = "0.53.0"
 val munitVersion           = "0.7.29"
 val munitCatsEffectVersion = "1.0.7"
+val kittensVersion         = "3.0.0-M4"
 
-ThisBuild / tlBaseVersion       := "0.35"
-ThisBuild / tlCiReleaseBranches := Seq("main", "scala3")
-ThisBuild / crossScalaVersions  := Seq("2.13.8", "3.1.2")
+ThisBuild / tlBaseVersion       := "0.36"
+ThisBuild / tlCiReleaseBranches := Seq("main")
+ThisBuild / crossScalaVersions  := Seq("3.1.3")
```

Comment:
I guess we'll make observe use scala 3, right?

Reply:
That's what we are shooting for. If we need to make a Scala 2 version of this project, we can open a branch.